bot/skills/towel_mode.py

```diff
@@ -78,18 +83,31 @@ def check_for_reply(update: Update, context: CallbackContext):
     if minute is None:
         return
 
-    if update.effective_message.reply_to_message is not None and \
-            update.effective_message.reply_to_message.from_user.id == context.bot.get_me().id:
-        logger.info(f"remove user: {update.effective_user} from quarantine")
-        del (quarantine[update.effective_user.id])
-        quarantine_min[minute].remove(update.effective_user.id)
-        update.message.reply_text("Welcome to VLDC! 😼")
-    else:
-        logger.debug(f"delete msg from user: {update.effective_user} [quarantine]")
-        context.bot.delete_message(
-            update.effective_chat.id,
-            update.effective_message.message_id, 10
-        )
+    logger.debug(
+        f"delete msg from user: {update.effective_user} [quarantine]")
+    context.bot.delete_message(
+        update.effective_chat.id,
+        update.effective_message.message_id, 10
+    )
+
+
+@run_async
+def butt_press(update: Update, context: CallbackContext):
+    quarantine, quarantine_min = get_quarantine_and_quarantine_min(context)
+    minute = quarantine.get(update.effective_user.id)
+    if minute is None:
+        return
+    chat_id = update.effective_chat.id
+    user = update.effective_user
+    query = update.callback_query
+    logger.info(query.data)
+    if query.data == 42 or True:
```

Comment:
@C-Pro looks like this condition is always True

Reply:
yep. log shows that 42 is passed here, but condition fails. IDK maybe it comes as a string. Explicitly converted it to string